Makefile

```diff
@@ -13,7 +13,7 @@ export MONGO_TEST_DB ?= 127.0.0.1:27017
 all: build test
 
 clean:
-	rm bin/*
```

Comment:
See http://stackoverflow.com/questions/2670130/make-how-to-continue-after-a-command-fails